stable/database/values.schema.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "type": "object",
+    "required": [
+        "database"
+    ],
+    "properties": {
+        "database": {
+            "type": "object",
+            "required": [
+                "archiveType"
+            ],
+            "properties": {
+                "archiveType": {
+                    "type": "string",
+                    "pattern": "^(|lsa)$"
```

Comment:
Is it possible to use [`enum`](https://json-schema.org/understanding-json-schema/reference/enum) instead?

That might generate a better error message and seems more consistent with how we are handling the property.

Reply:
good point - tested and it works! Adjusted code.